core/templates/pages/exploration-player-page/exploration-player-page.module.ts

```diff
@@ -30,7 +30,6 @@ import { LearnerViewInfoComponent } from './layout-directives/learner-view-info.
 import { MaterialModule } from 'modules/material.module';
 import { RefresherExplorationConfirmationModal } from './modals/refresher-exploration-confirmation-modal.component';
 import { ExplorationPlayerPageComponent } from './exploration-player-page.component';
-import { LessonInformationCardModalComponent } from './templates/lesson-information-card-modal.component';
```

Comment:
Please help me understand why we have to remove this modal from this module and import it to a common module.

Reply:
The `LessonInformationCardModalComponent` is used in the exploration player footer but is directly imported in only the exploration player and not the preview. So I had to move this component to the shared module which is used in both the preview and main player.

Reply:
Yes this makes sense, add another proof where the exploration player page is also working with the updated code.
Thanks

Reply:
Updated